src/PushwooshMessage.php

```diff
@@ -248,4 +252,20 @@ public function useRecipientTimezone()
 
         return $this;
     }
+
+    /**
+     * Enable or disable silent notifications for both Android and iOS platforms.
+     *
+     * @param bool $silent
+     * @return $this
+     */
+    public function silent(bool $silent = true)
+    {
+        if ($silent) {
+            $this->androidSilent = 1;
+            $this->iosSilent = 1;
+        }
```

Comment:
Otherwise `silent(true)->silent(false)` wil still set silent to `1`.
```suggestion
        if ($silent) {
            $this->androidSilent = 1;
            $this->iosSilent = 1;
        } else {
            $this->androidSilent = null;
            $this->iosSilent = null;
        }
```